libs/x-content/build.gradle

```diff
@@ -47,3 +60,24 @@ tasks.named("dependencyLicenses").configure {
 }
 
 tasks.named("jarHell").configure { enabled = false }
+
+File generatedResourcesDir = new File(buildDir, 'generated-resources')
+def generateProviderManifest = tasks.register("generateProviderManifest") {
+  File manifestFile = new File(generatedResourcesDir, "MANIFEST.TXT")
```

Comment:
I actually named it manifest since it sort of parallels the jar manifest. I'm not tied to the name, though. I just don't have a better one. Do you have a suggestion?

Reply:
Something with "embedded" in the name?

Reply:
I pushed "LISTING.TXT", wdyt?

Reply:
Works for me.